science/digital_rf/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           cxx11 1.1
+PortGroup           python 1.0
+
+github.setup        MITHaystack digital_rf 2.6.0
+maintainers         @MITHaystack {mit.edu:rvolz @ryanvolz} {mit.edu:swoboj @jswoboda}
+description         Read, write, and interact with data in the Digital RF and Digital Metadata formats.
+long_description    Tools to store RF and metadata for scientific engineering research projects.
+categories          science
+license             BSD
+platforms           darwin
+dist_subdir         digital_rf
+homepage            https://github.com/MITHaystack/digital_rf
```

Comment:
`homepage` is already set because you are using `github.setup` and you need not list it here.